.travis.yml

```diff
@@ -19,6 +19,7 @@ env:
     - CCACHE_COMPRESS=1
     - CC="clang"
     - CXX="clang++"
+    - HOST_OS="$(if [ "$TRAVIS_OS_NAME" == "osx" ]; then echo "darwin"; else echo "$TRAVIS_OS_NAME")"
```

Comment:
You've forgotten the `; fi` on the end of this if-block (assuming TravisCI even supports bash-style expressions in the yml file?)